MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import MapboxCoreNavigationObjC
```

Comment:
Could you explain why we sometimes use MapboxCoreNavigation and sometimes MapboxCoreNavigationObjC? Why the ObjC?

Reply:
Objective-C & Swift can not co-exist in a single swift package target. Therefore a separate target for the objective-C components is created.

Reply:
Ok got it.  Would this mean, since we are in a swift file here, that we should rather import the swift dependency?

Sorry if this is a stupid question :)

Reply:
It doesn't matter if you import it from Objective-C or Swift. If this file uses something that is defined in the Objc package, it should import the Objc package.